jvb/src/main/kotlin/org/jitsi/videobridge/transport/udp/UdpTransport.kt

```diff
@@ -93,12 +92,21 @@ class UdpTransport @JvmOverloads @Throws(SocketException::class, UnknownHostExce
             } catch (sce: SocketException) {
                 logger.info("Socket closed, stopping reader")
                 break
-            } catch (e: IOException) {
+            } catch (e: Exception) {
                 logger.warn("Exception while reading ", e)
+                stats.exceptionOccurred()
+                continue
             }
+
             val now = clock.instant()
             stats.packetReceived(packet.length, now)
-            incomingDataHandler?.dataReceived(buf, packet.offset, packet.length, now) ?: stats.incomingPacketDropped()
+            try {
+                incomingDataHandler?.dataReceived(buf, packet.offset, packet.length, now)
+                    ?: stats.incomingPacketDropped()
+            } catch (e: Exception) {
+                stats.exceptionOccurred()
+                logger.warn("Exception while handling:", e)
```

Comment:
The punctuation of the two exception warnings is different (one has a space and one has a colon), but that's not a big enough deal to block approval.